controllers/managedocs_controller.go

```diff
@@ -47,8 +47,12 @@ type ManagedOCSReconciler struct {
 	managedOCS *v1.ManagedOCS
 }
 
-// +kubebuilder:rbac:groups=ocs.openshift.io,resources=managedocs,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=ocs.openshift.io,resources=managedocs/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=managedocs,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=storageclusters,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=managedocs/status,verbs=get;update;patch
+
+// Add necessary rbac permissions for managedocs finalizer in order to set blockOwnerDeletion.
+// +kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=managedocs/finalizers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
I am not sure `namespace=system` is the correct place for these RBAC

Reply:
Based on kubebuilder markers, the rules will set to Role only when a namespace is mentioned else the rules will set to ClusterRole. The namespace value will be replaced with openshift-storage by kustomize, thus leading to creation of roles in openshift-storage namespace.

Reply:
Have you verified this behaviour. If so I will approve the changes

Reply:
yes, verified using `make deploy` .

Reply:
Can you also check what is written to the bundle directory after `make manifests`?

Reply:
Based on my observation, `make manifests` doesn't modify any contents in bundle directory, it mainly updates rbac files in case of any changes.